src/GitHub.VisualStudio/Services/ConnectionManager.cs

```diff
@@ -169,7 +209,13 @@ void SaveConnectionsToCache()
 
             var cache = new CacheData();
             cache.connections = Connections.Select(conn =>
-                new ConnectionCacheItem { HostUrl = conn.HostAddress.WebUri, UserName = conn.Username });
+                new ConnectionCacheItem
+                {
+                    HostUrl = conn.HostAddress.WebUri,
+                    UserName = conn.Username,
+                    /* Repositories = conn.Repositories.Select(x =>
```

Comment:
Ditto this comment. Comments like this make me think the code was commented out for testing and that the developer forgot to uncomment it. :stuck_out_tongue: